core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay

```diff
@@ -113,8 +113,12 @@ registry.ci.openshift.org/ocp-kni/4.16:cluster-group-upgrades-operator-bundle qu
 registry.ci.openshift.org/ocp-kni/4.16:cluster-group-upgrades-operator-recovery quay.io/openshift-kni/cluster-group-upgrades-operator-recovery:4.16 quay.io/openshift-kni/cluster-group-upgrades-operator-recovery:4.16.0 quay.io/openshift-kni/cluster-group-upgrades-operator-recovery:latest
 registry.ci.openshift.org/ocp-kni/4.16:cluster-group-upgrades-operator-precache quay.io/openshift-kni/cluster-group-upgrades-operator-precache:4.16 quay.io/openshift-kni/cluster-group-upgrades-operator-precache:4.16.0 quay.io/openshift-kni/cluster-group-upgrades-operator-precache:latest
 # lifecycle-agent-operator
-registry.ci.openshift.org/ocp-kni/4.14:lifecycle-agent-operator quay.io/openshift-kni/lifecycle-agent-operator:4.14 quay.io/openshift-kni/lifecycle-agent-operator:4.14.0 quay.io/openshift-kni/lifecycle-agent-operator:latest
-registry.ci.openshift.org/ocp-kni/4.14:lifecycle-agent-operator-bundle quay.io/openshift-kni/lifecycle-agent-operator-bundle:4.14 quay.io/openshift-kni/lifecycle-agent-operator-bundle:4.14.0 quay.io/openshift-kni/lifecycle-agent-operator-bundle:latest
+registry.ci.openshift.org/ocp-kni/4.14:lifecycle-agent-operator quay.io/openshift-kni/lifecycle-agent-operator:4.14 quay.io/openshift-kni/lifecycle-agent-operator:4.14.0
```

Comment:
Do we need to keep 4.14 here?

Reply:
We will have 4.14 eventually. I assume this doesn't do anything since we don't build 4.14 images for now.